website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -105,6 +105,8 @@ The following arguments are supported:
 
 * `node_taints` - (Optional) A list of Kubernetes taints which should be applied to nodes in the agent pool (e.g `key=value:NoSchedule`). Changing this forces a new resource to be created.
 
+~> **Note:** Node Taints control the behaviour of the Node Pool, as such they must be specified and reconciled as required.
```

Comment:
this is true of all attributes? perhaps it's worth a comment in the schema instead?

```suggestion
```